### DISCUSSION TITLE
2022-26: 哑巴英语首次开口

### DISCUSSION THREAD

**Discussion by giscus** [Xuanwo/blog#364] [Announcements]
# 2022-26: 哑巴英语首次开口

An infrastructure engineer, focused on distributed storage system

https://xuanwo.io/reports/2022-26/

**Comment by STRRL** [comment]
> 感觉语音交流很有意思，以后希望能多参加一些类似的访谈节目，免得自己长期在家语言沟通能力退化。

我们可以也自己录播客玩!

**Comment by mxyq** [comment]
俺推荐一个语法书《英语语法最快过关》.刘戈锋，这本语法书能够帮助读者快速建立语法结构。
